news/2023/2023-12-16-nat-gazette-dec-2023.md

```diff
@@ -0,0 +1,107 @@
+- **osu!taiko:** Relaxed the rest moment guideline for Kantan and Futsuu difficulties from 16/1—20/1 to 32/1—36/1.
+- **osu!catch:** Changed the CS and HP guidelines for Rain and Overdose difficulties respectively, and an extra guideline on how to use higher-snapped hyperdashes was added.
+- **osu!mania:** Key modes over 10 keys are now rankable so long they follow an allowed [playstyle](/wiki/Beatmapping/osu!mania_10K_plus_playstyles).
+
+### Within the Beatmap Nominators
+
+#### Changes to activity requirements
+
+Beatmap Nominators rejoining in good terms are now expected to nominate their mode's [minimum activity requirements](/wiki/People/Beatmap_Nominators/Rules#requirements) in the first 30 days since their rejoin. Make sure to take this into consideration before asking to join back into the team!
+
+Additionally, we've decided to remove the probation-specific activity requirements and allow NAT members again to apply their own judgment when deciding if a BN needs to have their probation extended for the lack of enough nominations to evaluate. Do note that the regular minimum activity requirements still apply!
+
+#### Joining & rejoining
+
+All modes will now require **150 [kudosu!](/wiki/Modding/Kudosu)** to be able to apply, bringing osu!'s requirement in line with the other modes. Moreover, Beatmap Nominators leaving in good terms can rejoin the team instantly **up to one year** (as opposed to 6 months previously) after their departure! Members who got kicked for their low activity however, will now be able to submit a new application right after their removal, instead of having to wait 30 days.
+
+Finally, cooldowns for failed applications have been reduced all across the board! The standard length is now reduced from 90 days to **60 days**, with reduced cooldowns being only **30 days** now. Failing an application will not be as punishing as it used to be, so don't be afraid to [apply to join the Beatmap Nominators today](https://bn.mappersguild.com/bnapps)!
+
+#### Better visibility for BN preferences
+
+Beatmap Nominators can now set their map and song preferences in the [BN Management website](https://bn.mappersguild.com), alongside their custom request criteria in a much more detailed way. This will make it easier for users to know who to request a map to by parsing through the list of open members and their respective preferences.
+
+Simply click on any user in the [home page](https://bn.mappersguild.com/home) to see their detailed request preferences as seen below:
+
+![BN request info](/wiki/shared/news/2023-12-16-nat-gazette-dec-2023/request.png)
+
+### Within the Nomination Assessment Team
+
+#### Evaluation changes
+
+Activity evaluations will (mostly) be automated. The consensus and feedback will be set automatically, but these will still need a NAT member to check them before they can be sent. Additionally, a Beatmap Nominator's evaluation cycle can no longer be extended past 3 months.
+
+#### The Impact of our actions
```

Comment:
do we really need to capitalise "impact"?

Reply:
not really